types/utils.go

```diff
@@ -125,3 +128,24 @@ func ParseLengthPrefixedBytes(key []byte, startIndex int, sliceLength int) ([]by
 
 	return byteSlice, endIndex
 }
+
+func MarshalGenesisStateToJSON(gs map[string]proto.Message, cdc codec.JSONCodec, indent bool) ([]byte, error) {
+	genStateJson := map[string]json.RawMessage{}
+	var err error
+	for k, v := range gs {
+		if v != nil {
+			genStateJson[k], err = cdc.MarshalJSON(v)
+			if err != nil {
+				return nil, err
+			}
+		} else {
+			genStateJson[k] = []byte("{}")
+		}
+	}
+
+	if indent {
+		return tmjson.MarshalIndent(genStateJson, "", "  ")
```

Comment:
Any reason to prefer tmjson over json? I think the outputs will be exactly the same here.

I personally feel there's no need for tmjson in the SDK.

Reply:
No idea tbh, I just used it because in many places we were using it. I can switch to json